app/models/crop.rb

```diff
@@ -52,11 +52,19 @@ class Crop
   has_many :varieties, class_name: 'Crop', inverse_of: :parent
   belongs_to :parent, class_name: 'Crop', inverse_of: :varieties
 
+  has_and_belongs_to_many :companions,
+                          class_name: 'Crop',
+                          inverse_of: :companions
+
```

Comment:
Hey @RickCarlino, I thought about doing it this way, but I didn't really like the look of having a suggested_companions and a companions object on the crop element. It seemed like redundant information and might be confusing.

Would companions and suggested_companions always be the same? No right?

If **tomato** is a `companion` to **corn**, then **corn** wouldn't be a `companion` to **tomato**, but rather a `suggested_companion`? And then you'd always have to merge the two? Which I guess could be done through the serializer, but that seems wrong as well. 

Or at least, that's how I understood it.